From 9ae386653c321c8ddc10fad5ea88f32ebb3d3ffe Mon Sep 17 00:00:00 2001
From: aavit <eirik.aavitsland@digia.com>
Date: Fri, 22 Nov 2013 15:04:23 +0100
Subject: [PATCH] Recognize newer libmng versions in config test

libmng 2.0.x has been released and is compatible and usable, but since
it no longer provides a VERSION_MAJOR macro, the config test would fail.

Task-number: QTBUG-34894
Change-Id: I106aa258de0851af01d1bb016c2971dd8e30fd24
Reviewed-by: Liang Qi <liang.qi@digia.com>
---
 config.tests/libmng/libmng.cpp |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/config.tests/libmng/libmng.cpp b/config.tests/libmng/libmng.cpp
index 9def33e..fc3e693 100644
--- a/config.tests/libmng/libmng.cpp
+++ b/config.tests/libmng/libmng.cpp
@@ -46,9 +46,11 @@ int main(int, char **)
     mng_handle hMNG;
     mng_cleanup(&hMNG);
 
+#if defined(MNG_VERSION_MAJOR)
 #if MNG_VERSION_MAJOR < 1 || (MNG_VERSION_MAJOR == 1 && MNG_VERSION_MINOR == 0 && MNG_VERSION_RELEASE < 9)
 #error System libmng version is less than 1.0.9; using built-in version instead.
 #endif
+#endif
 
     return 0;
 }
-- 
1.7.1

